product_vision.md

```diff
@@ -0,0 +1,65 @@
+## Product Vision
+A world where blind and vision impaired people can independently and fully interact with the Windows Operating System and popular third party applications, enabling them to contribute to society, no matter their language, location or economic status.
+Blind and vision impaired people are empowered to control their own destiny and ensure this vision, through being able to significantly contribute to the technology that makes this possible.
+
+## Product description
+NVDA (NonVisual Desktop Access) is software that allows blind and vision impaired people to independently interact with the Windows Operating System and popular third party applications by outputting content in either synthetic speech or via a refreshable Braille Display.
+Developed by NV Access in partnership with a dedicated open source community, NVDA enables blind and vision impaired people to increase their chances at education and employment, and to make a contribution to society.
```

Comment:
This is perhaps more of a political question - but I am interested to know thoughts on rephrasing "make a contribution to society" to "participate within society". 
The phrase "contribute to society" is also used in the first sentence of the product vision.

The difference being an expectation of contributing / performing work, which might not be suitable to all users.
Lowering barriers for employment and education is great - but I've also heard user stories about NVDA aiding participation in society in other ways, e.g. social platforms.

Maybe this can be a separate added sentence rather than rephrasing this one, mentioning that NVDA aims to enable users to participate in society, outside education and employment.

Reply:
I strongly agree with @seanbudd. Participation is related to something much more active than productive contribution, who may not be possible for everyone due to expectations and considerations about what productivity it. Contribution sounds like to give something to a given social structire, and participation seems more precise meaning action,not just accepting and helping.

Reply:
Some good points here.
As context, ever since the beginning Jamie and I have always said "contribute to society" as we believe that everyone has a right to contribute to society, though some may require some extra tools to do so.
Also note that the statement says "allow", I.e. we are allowing people to contribute if they wish, not forcing or expecting them to.
"Participate" may be a more acceptable word. What are your thoughts @jcsteh ? I think both have their own valid need. "participate in and or contribute to society" is what I would want, but perhaps "participate" covers both enough.

Reply:
This is an interesting question. I've always used the word "contribute" because I strongly believe that everyone has a right but also a responsibility to contribute to society in some way. Historically, even where people with disabilities might not have been excluded from participating in certain aspects of society, there were more often excluded from contributing because they didn't have the right tools or because they weren't given the right opportunities. I believe this also created a situation where this norm became so firmly established that even where opportunities and tools became available, some people with disabilities didn't contribute (and perhaps didn't even feel they should have to) even though they could have. Given the right tools and opportunities, I don't think anyone should be excluded from this responsibility or the rights that come with it. Reflecting on this, I do realise this stance comes to some extent from a position of privilege. Just because I have opportunities doesn't mean that everyone does.

Also, it's worth noting that contributing to society doesn't necessarily mean traditional contribution. You don't have to work a paid job and pay taxes to contribute. Perhaps you volunteer where it's needed most, etc.

All of that said, I think it's reasonable to say "participate in and contribute to society". I don't think I'd want to drop contribute to - I still think this is important for all the reasons outlined above - but adding "participate" makes it clear that we think both are important. As @MichaelDCurran noted, "enables" also makes it clear that we want to enable these things, but that we can't force or require our users to do them.